content/blog/2019-02-06-interview-with-hostinger.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Interview with Hostinger
+created_at: 2019-02-06
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Donatas Abraitis
+from Hostinger talks about their monitoring journey.*
+
+## Can you tell us about yourself and what Hostinger does?
+
+I’m Donatas Abraitis, a systems engineer at
+[Hostinger](https://www.hostinger.com/). Hostinger is the hosting company as
+the name implies. We have around 30M clients since 2004 including
+the [000webhost.com](https://www.000webhost.com/) project - free web hosting provider.
+
+## What was your pre-Prometheus monitoring experience?
+
+When Hostinger was quite a small company, only Nagios, Cacti and Ganglia
+existed at that time in the market as open source monitoring tools. This is
```

Comment:
Maybe `as popular open source...`, there are lots of other tools. Maybe also drop `only` from above.